crates/telio-nurse/src/qos.rs

```diff
@@ -274,28 +285,44 @@ impl Analytics {
         for key in sorted_public_keys {
             if let Some(node) = nodes.get(key) {
                 // RTT
-                output.rtt.push_str(&Analytics::percentile_histogram(
-                    &node.rtt_histogram,
-                    buckets,
-                ));
+                if node.rtt_histogram.entries() > 0 {
```

Comment:
We can extract out ln 288:296 into a function and repeat it for other rtts.

Reply:
Fixed